docs/MOM_parameter_doc.layout

```diff
@@ -46,6 +46,10 @@ NJPROC = 30                     !
                                 ! in MOM_memory.h at compile time.
 LAYOUT = 36, 30                 !
                                 ! The processor layout that was actually used.
+AUTO_IO_LAYOUT_FAC = 0          ! default = 0
+                                ! When AUTO_MASKTABLE is enabled, io layout is calculated by performing integer
+                                ! division of the runtime-determined domain layout with this factor. If the
+                                ! factor is set to 0 (default), the io layout is set to 1,1.
 IO_LAYOUT = 1, 1                ! default = 1
```

Comment:
It looks like `IO_Layout` is not used? Can we delete it? It is determined by `AUTO_IO_LAYOUT_FAC`? 

Per https://gmd.copernicus.org/articles/13/1885/2020/ it looks like the default is the right value, but is performance impacted if you increase AUTO_IO_LAYOUT_FAC ?

Reply:
`MOM_parameter_doc.layout` is an auto-generated file, so there is no need to manually delete any parameters in it. The I/O layout has not been tuned yet, but it will definitely change when varying `AUTO_IO_LAYOUT_FAC`.